hack/check-commit-message.sh

```diff
@@ -4,11 +4,12 @@ set -o nounset
 
 commit_file=${1}
 commit_message="$(cat ${commit_file})"
-valid_commit_regex='^(NO-ISSUE|Merge|Revert|((Bug |[A-Z]+-|#)[0-9]+:))'
+valid_commit_regex='^(NO-ISSUE|Merge|Revert|([A-Z]+-[0-9]+:))'
+valid_github_regex='^((Fixes|fixes|Closes|closes): #[0-9]+)'
 
 error_msg="""Aborting commit: ${commit_message}
 ---
-Your commit message should start with a JIRA issue ('JIRA-1111'), a GitHub issue ('#39'), or a BugZilla issue ('Bug 123')
+Your commit message should start with a JIRA issue ('JIRA-1111') or a GitHub issue ('Fixes:#39')
```

Comment:
```suggestion
Your commit message should start with a JIRA issue ('JIRA-1111') or a GitHub issue ('Fixes #39')
```
 I propose this format instead.

Reply:
Done